extensions/guacamole-vault/modules/guacamole-vault-ksm/src/main/java/org/apache/guacamole/vault/ksm/secret/KsmSecretService.java

```diff
@@ -338,12 +332,59 @@ public Map<String, Future<String>> getTokens(UserContext userContext, Connectabl
                 addRecordTokens(tokens, "KEEPER_GATEWAY_",
                         ksm.getRecordByHost(filter.filter(gatewayHostname)));
 
+            // Retrieve and define domain tokens, if any
+            String domain = parameters.get("domain");
+            String filteredDomain = null;
+            if (domain != null && !domain.isEmpty()) {
+                filteredDomain = filter.filter(domain);
+                addRecordTokens(tokens, "{KEEPER_DOMAIN_",
```

Comment:
This should be `KEEPER_DOMAIN_`, not `{KEEPER_DOMAIN_`.

Reply:
Oops!